articles/storage/blobs/upgrade-to-data-lake-storage-gen2.md

```diff
@@ -80,7 +80,7 @@ Renaming a blob is far more efficient because client applications can rename a b
 
 ## Impact on costs
 
-Storage costs aren't impacted, but transactions costs are impacted. Use these pages to assess compare costs. 
+While the operation of upgrade to gen2 itself is free and storage costs aren't impacted, but transactions costs are impacted. Use these pages to assess compare costs. 
```

Comment:
I would rephrase slightly - Can we just replace everything before "Use these pages" with "There is no cost to perform the upgrade. After you upgrade, the cost to store your data doesn't change, but the cost a transaction does change." Thoughts?

Reply:
That sounds good. Now that I read it, mine sounds grammatically a bit off. Will make those changes.